docs/native-modules-setup.md

```diff
@@ -176,12 +176,12 @@ Next you need to create a YAML file for GitHub Actions, the basic steps are:
         node-version: '12.9.1'
 
     - name: Setup MSBuild
-      uses: microsoft/setup-msbuild@v1.0.0
+      uses: microsoft/setup-msbuild@v1
       with:
         vs-version: 16.5
 
     - name: Setup NuGet
-      uses: NuGet/setup-nuget@v1.0.2
+      uses: NuGet/setup-nuget@v1

```

Comment:
If this is targeting 0.64+, you should be able to delete this now, we use msbuild's built-in nuget support now

Reply:
Ah so we can remove the setup Nuget all together for 64+ then?

Reply:
yes, see #6548

Reply:
Changes made in PR #352